htdocs/includes/odtphp/odf.php

```diff
@@ -161,7 +161,8 @@ public function setVars($key, $value, $encode = true, $charset = 'ISO-8859')
 	 */
 	public function convertVarToOdf($value, $encode = true, $charset = 'ISO-8859')
 	{
-		$value = $encode ? htmlspecialchars($value) : $value;
+        $value = dol_html_entity_decode($value, ENT_COMPAT);
```

Comment:
Can you confirm this:  This means the parameter $encode will not be used anymore ?